code/src/index.css

```diff
@@ -1,13 +1,245 @@
     monospace;
 }
+
+/* Movie cards styling*/
+
+.movie-cards-container { 
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+  flex-flow: wrap;
+}
+
+.movie-card {
+  flex-basis: 25%;
+}
+
+ .movie-card img {
+   width: 100%;
+   margin: 0;
+}
+.overlay-container {
+  position: relative;
+}
+
+.text-overlay { 
+  position: absolute;
+  bottom: 0;
+  color: #fff;
+  background: rgba(0, 0, 0, 0.742);
+  width: 100%;
+  height: 100%;
+  transition: 0.3s ease;
+  opacity: 0;
+}
+
+.text-overlay h1 { 
+  position: absolute;
+  bottom: 50px;
+  margin-left: 20px;
+  font-size: 30px;
+}
+
+.text-overlay p {
+  position: absolute;
+  bottom: 0;
+  padding: 20px;
+  font-size: 17px;
+}
+
+.text-overlay:hover {
+  opacity: 1;
+}
+
+/* Movie pages Styling */
+.movie-page { 
+  display: flex;
+}
+
+.movie-page-details {
+  display: flex;
+  align-items: flex-end;
+  padding: 30px;
+}
+
+.movie-page {
+  width: 100%;
+  background-size: cover;
+  min-height: 100vh;
+}
+
+.movie-page-details {
+  color: #fff;
+  padding: 20px;
+}
+
+.movie-poster {
+  width: 50%;
+  border: 5px solid #fff;
+}
+
+h2 {
+  margin-left: 10px;
+  margin-bottom: 0;
+}
+.movie-rating {
+  color:#dc1035;
+  margin-left:10px;
+  margin-top:0;
+}
+
+.movie-overview { 
+  font-size: 14px;
+  padding-left: 10px;
+  margin: 0;
+}
+
+.details-container { 
+  width: 100%;
+}
+
+.back-link { 
+  position: absolute;
+  text-decoration: none;
+  font-size: 20px;
+  color: #fff;
+  margin: 50px;
+  padding-left:30px;
+  font-weight: bold;
+  -webkit-transform: perspective(1px) translateZ(0);
+  transform: perspective(1px) translateZ(0);
+  box-shadow: 0 0 1px rgba(0, 0, 0, 0);
+  -webkit-transition-duration: 0.3s;
+  transition-duration: 0.3s;
+  -webkit-transition-property: transform;
+  transition-property: transform;
+}
+  .back-link:hover, .back-link:focus, .back-link:active {
+  -webkit-transform: translateX(8px);
+  transform: translateX(8px);
+  }
+  
+  .back-arrow { 
+    transform: translate(-40%, 29%);
+    width: 30px;
+    height: 30px;
+  }
+
+  /* NOT FOUND */
+
+  .not-found { 
+    width: 100%;
+    color: white;
+    height: 100vh;
+    text-align: center;
+  }
+
+  h1, h2, h3 {
+    font-size: 20px;
+    margin-bottom: 0px;
+  }
+
+  h4 {
+    font-size: 20px;
+    margin-top: 0;
+    color: #feb75a;
+    transform: translateX(-65%);
+  }
+
+  .bird-gif {
+    width: 300px;
+    height: 200px;
+    margin: 20px;
+  }
+
+
+  /* FOOTER */
+
+footer {
+  position: absolute;
+  height: 150px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-wrap: wrap;
+  padding: 0px;
+  background-color: #ffffff67;
+  width: 100%;
+}
+
+footer p {
+  width: 100%;
+  text-align: center;
+  font-size: 17px;
+  color: rgb(255, 255, 255);
+  margin: 6px;
+}
+
+footer a:link {
+  color: rgb(245, 108, 156);
+}
+
+footer a:visited {
+  color: rgb(130, 149, 228);
+}
+
+footer img {
+  width: 50px;
+}
+
+
+@media (min-width: 1024px) {
+
```

Comment:
please review this: title and release date text overlaps when not displaying on desktop. i'm using chrome on my mac.
<img width="192" alt="Screenshot 2020-11-11 at 7 20 49 PM" src="https://user-images.githubusercontent.com/64912576/98849227-2feed980-2453-11eb-8755-dc506e8451ac.png">

Reply:
Fixed! at least over here looks good now, let me know if it doesn't show properly on yours 😊

Reply:
yay! it works now!

Reply:
Awesome! Thank you  so much for the review and for pointing it out, had no idea what happened there! 😅